jujugui/static/gui/src/app/views/state.js

```diff
@@ -117,7 +117,7 @@ YUI.add('juju-app-state', function(Y) {
       @param {Object} state The current state object.
     */
     dispatch: function(state) {
-      var sections = ['app', 'sectionA', 'sectionB'];
+      var sections = ['app', 'sectionA', 'sectionB', 'sectionC'];
```

Comment:
I still really dislike having these be named, and also to be named twice, but that's out of scope.